docs/resources/README.md

```diff
@@ -0,0 +1,69 @@
+# Resources
+
+Quick links:
+
+* [Building your own resource detector](#resource-detector)
+
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+is the immutable representation of the entity producing the telemetry. If no
+`Resource` is explicitly configured, the
+[default](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#semantic-attributes-with-sdk-provided-default-value)
+is to use a resource indicating this
+[Service](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#service)
+and [Telemetry
+SDK](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#telemetry-sdk).
+The `ConfigureResource` method on `TracerProviderBuilder` can be used to
+configure the resource on the provider. `ConfigureResource` accepts an `Action`
+to configure the `ResourceBuilder`. Multiple calls to `ConfigureResource` can be
+made. When the provider is built, it builds the final `Resource` combining all
+the `ConfigureResource` calls. There can only be a single `Resource` associated
+with a provider. It is not possible to change the resource builder *after* the
+provider is built, by calling the `Build()` method on the
+`TracerProviderBuilder`.
+
+`ResourceBuilder` offers various methods to construct resource comprising of
+multiple attributes from various sources. Examples include `AddTelemetrySdk()`
+which adds [Telemetry
+Sdk](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#telemetry-sdk)
+resource, and `AddService()` which adds
+[Service](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#service)
+resource. It also allows adding `ResourceDetector`s.
+
+Follow [this](#resource-detector) document
+to learn about writing custom resource detectors.
+
+The snippet below shows configuring the `Resource` associated with the provider.
```

Comment:
this section is tricky, as it is tied to Tracing signal.. We need to document all three with SDK.CreateProvider and the OTel.Ext.Hosting style of configuration of resource.

The env variables part is good for all three.

Reply:
@luizhlelis See if you can fix this part too. If not, we can still merge, and do some follow ups later.